docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -17,63 +25,55 @@ Distributing collections involves three major steps:
 
 .. _config_distribution_server:
 
-Configuring your distribution server or servers
-================================================
+Initial configuration of your distribution server or servers
+============================================================
+
+Configure a connection to one or more distribution servers so you can publish collections there. You only need to configure each distribution server once. You must repeat the other steps (building your collection tarball, preparing to publish, and publishing your collection) every time you publish a new collection or a new version of an existing collection.
 
-1. Get a namespace on each distribution server you want to use (Galaxy, private Automation Hub, Red Hat Automation Hub).
+1. Create a namespace on each distribution server you want to use (Galaxy, private Automation Hub, Red Hat Automation Hub).
 2. Get an API token for each distribution server you want to use.
 3. Specify the API token for each distribution server you want to use.
 
 .. _get_namespace:
 
-Getting a namespace
--------------------
+Creating a namespace
+--------------------
 
-You need a namespace on Galaxy and/or Automation Hub to upload your collection. To get a namespace:
+You must have a namespace on each distribution server (Galaxy, Red Hat Automation Hub, private Automation Hub) to upload your collection. To create a namespace:
```

Comment:
Technically you don't need to create a namespace on community galaxy. By default your namespace is your username.

Reply:
Hm, so as long as you have a user account on Galaxy, you also have a namespace? Thanks, I'll add that in.

Reply:
Changed in 2b84951fc61422551f159ccf5df686beff5fb23e. I hope this is clear now.